### ISSUE TITLE
[Polish] Weird lines on congratulations dialog in Edge

### ISSUE THREAD

**Issue by poorvasingal** [code-dot-org/dance-party#342]
![image](https://user-images.githubusercontent.com/16227893/48392289-7cb24500-e6bf-11e8-98e4-5bc2b5e1042d.png)

**Comment by ryansloan** [comment]
Very Mondrian. 
@poorvasingal Does this repro only on dance or on other finish dialogs as well?

**Comment by poorvasingal** [comment]
Doesn't seem specific to Dance Party.

**Comment by boatmarina** [comment]
@poorvasingal i don't repro this on edge in windows.  do you still repro this and is it every time?

**Comment by boatmarina** [comment]
if you don't repro this, let's won't fix this.

**Comment by poorvasingal** [comment]
Yeah I repro'd this today, the first time I tried it. I definitely don't think it's a big deal if we won't fix. Just wanted to over-log instead of under-log.
